### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IanHakes-okta**
Update yarn release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ There are currently 2 parts to the site, the content and the theming/plugins.
 ### Requirements
 
 - [Node](https://nodejs.org/en/download/): 8+
-- [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable): 1.15+
+- [Yarn](https://yarnpkg.com/lang/en/docs/install/): 1.7 or later
```
Comment (bretterer) [inline README.md]: Did you confirm that everything still builds and the dev server runs with yarn 1.7?

Comment (IanHakes-okta) [reply README.md]: I have not, no - I was making the change based on your slack comment. I don't have a CentOS / Strap environment to test in.

**Comment by IanHakes-okta** [comment]
@donwoods-okta - please test that `yarn dev` and `yarn build` work on your 1.7 environment, and we can update the documentation.

**Comment by donwoods-okta** [comment]
Alas, I can neither confirm nor deny that 1.7 works in my CentOS environment.  I ran into a problem that turned out to be due to the ~/.npmrc file, but while trying things to get past that issue I upgraded my yarn to 1.15.2.

**Pull request closed without merging**